diff --git a/dist/request.js b/dist/request.js
index bde03bb5daa16e077ea271d9ceccd3b17c253e40..cea71836c20ce7c5aac391a30b5cdbb904af6c0e 100644
--- a/dist/request.js
+++ b/dist/request.js
@@ -29,33 +29,10 @@ const inNode = typeof process !== "undefined" &&
     process.versions != null &&
     process.versions.node != null;
 async function get(url) {
-    if (inBrowser) {
-        const response = await window.fetch(url);
+        const response = await fetch(url);
         return {
             status: response.status,
             data: await response.json(),
         };
-    }
-    else if (inNode) {
-        const getter = url.startsWith("https")
-            ? await Promise.resolve().then(() => __importStar(require("https")))
-            : await Promise.resolve().then(() => __importStar(require("http")));
-        return await new Promise((res, rej) => {
-            const request = getter.get(url, (response) => {
-                let data = "";
-                response.on("error", rej);
-                response.on("data", (chunk) => (data += chunk.toString()));
-                response.on("end", () => res({
-                    status: response.statusCode,
-                    data: JSON.parse(data),
-                }));
-            });
-            request.on("error", rej);
-            request.end();
-        });
-    }
-    else {
-        throw new Error("Unknown environment");
-    }
 }
 exports.get = get;
